tests/foreman/cli/test_foremantask.py

```diff
@@ -0,0 +1,47 @@
+"""Tests for the ``foreman_tasks/api/v2/tasks`` path.
+
+:Requirement: Foremantask
+
+:CaseAutomation: Automated
+
+:CaseLevel: Component
+
+:CaseComponent: TasksPlugin
+
+:Assignee: pondrejk
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+import pytest
+
+
+@pytest.mark.stubbed
+@pytest.mark.tier1
```

Comment:
It seems like for both tests we are looking at how satellite installer interacts with the system. Therefore I would say both are `tier3`. More detail here or could ask around for what would fit better, but it doesn't seem like either of these tests would be CRUD: https://github.com/SatelliteQE/robottelo/blob/master/pytest_plugins/markers.py#L11